### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by SolivanLau**
### Nav
- added **aria-current = "page"** to relative links with styles only applying to header navigation links

### Team
- removed socials as requested, from team cards and team modal
- web copy adjustments US to us 🗽

### Events
- section padding adjusted, img max widths adjusted
- button hover state adjusted w new global class **btn-hover-drk**

### CODE REVIEW

File: css/events.css [hunk 1]
```diff
@@ -10,6 +10,9 @@ header {
 	min-height: auto;
 }
 
+section {
+	padding: var(--spacing-lg) 0;
+}
```
Comment (SolivanLau) [inline css/events.css:15]: applying on events and event-single pages only

File: about.html [hunk 2]
```diff
@@ -70,7 +70,7 @@
 
 							<!-- Page Links -->
 							<ul id="nav-links" class="nav-links">
-								<li><a href="/about.html">About Us</a></li>
+								<li><a href="/about.html" aria-current="page">About Us</a></li>
```
Comment (maxharrisnet) [inline about.html:73]: This aria stuff is great, thanks!

File: css/events.css (hunk 1 above)
Comment (maxharrisnet) [reply css/events.css:15]: Looks better, thanks

**Review by maxharrisnet — APPROVED** [review]
LGTM 🚀

**Pull request merged**